src/Orleans.Runtime/Silo/Silo.cs

```diff
@@ -454,11 +457,18 @@ public async Task StopAsync(CancellationToken cancellationToken)
 
             if (stopAlreadyInProgress)
             {
-                logger.Info(ErrorCode.SiloStopInProgress, "Silo termination is in progress - Will wait for it to finish");
-                var pause = TimeSpan.FromSeconds(1);
-                while (!this.SystemStatus.Equals(SystemStatus.Terminated))
+                if (logger.IsEnabled(LogLevel.Debug))
                 {
-                    logger.Info(ErrorCode.WaitingForSiloStop, "Waiting {0} for termination to complete", pause);
+                    logger.Debug(ErrorCode.SiloStopInProgress, "Silo shutdown in progress. Waiting for shutdown to be completed.");
+                }
+                var pause = TimeSpan.FromSeconds(1);                
+
+                while (!this.SystemStatus.Equals(SystemStatus.Terminated))
+                {                    
+                    if (logger.IsEnabled(LogLevel.Debug))
+                    {
+                        logger.Debug(ErrorCode.WaitingForSiloStop, "Silo shutdown still in progress...", pause);
```

Comment:
The pause parameter isn't being logged here, but it's being passed to the log call.